examples/text-classification/run_glue.py

```diff
@@ -207,7 +215,25 @@ def main():
     # download the dataset.
     if data_args.task_name is not None:
         # Downloading and loading a dataset from the hub.
-        datasets = load_dataset("glue", data_args.task_name)
+        if data_args.task_name == "mnli":
+            if data_args.mnli_compat_mode:
+                logger.warning(
+                    "The labels assignment is overwritten with the old assignment"
+                    "['contradiction', 'entailment', 'neutral'] from transformers<3.5.0."
+                )
+                if version.parse(datasets.__version__) < version.parse("1.3.0"):
+                    raise ImportError("Please update `datasets` to >1.3.0 to overwrite the labels.")
+                task_datasets = datasets.load_dataset(
+                    "glue", data_args.task_name, label_classes=["contradiction", "entailment", "neutral"]
```

Comment:
```suggestion
                    "glue", data_args.task_name, label_classes=["contradiction", "neutral" , "entailment"]
```
I believe this is the correct order!